runner/webserver.ts

```diff
@@ -179,9 +186,11 @@ Expected to find a ${mdFilenames.join(' or ')} at: ${pathToLocalWct}/
     }
 
     options.webserver._servers = servers.map(s => {
+      const npmParam = options.npm ? `?npm=true` : '';
       const port = s.server.address().port;
+      const url = `http://localhost:${port}${pathToGeneratedIndex}`;
       return {
-        url: `http://localhost:${port}${pathToGeneratedIndex}`,
+        url: `${url}${npmParam}`,
```

Comment:
In the initial URL generation there are no parameters and when `cli_browser` is added there's a check to add via ? or &